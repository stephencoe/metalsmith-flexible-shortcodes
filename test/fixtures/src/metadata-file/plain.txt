---
title: Plain
shortcodes: true
test: 'Hello'
---
This should be 'Hello':
[var test/]
