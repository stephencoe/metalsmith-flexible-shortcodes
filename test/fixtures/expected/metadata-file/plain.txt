This should be 'Hello':
Hello
